src/ejabberd_s2s.erl

```diff
@@ -370,7 +369,7 @@ choose_pid(From, Pids) ->
                 [] -> Pids;
                 Ps -> Ps
             end,
-    % Use sticky connections based on the JID of the sender (whithout
+    % Use sticky connections based on the JID of the sender (without
```

Comment:
Good catch. I wouldn't mind, if we're already touching this one, if we put the parenthesis in the line below, like
```erlang
    % Use sticky connections based on the JID of the sender
    % (without the resource to ensure that a muc room always uses the same connection)
```